cmd/gpu-feature-discovery/main.go

```diff
@@ -86,6 +86,12 @@ func main() {
 			Value:   "/etc/kubernetes/node-feature-discovery/features.d/gfd",
 			EnvVars: []string{"GFD_OUTPUT_FILE"},
 		},
+		&cli.StringFlag{
+			Name:    "imex-nodes-config-file",
+			Usage:   "Path to the IMEX ",
```

Comment:
```suggestion
			Usage:   "Path to the IMEX domain configuration file",
```

Reply:
To be clear -- the "nodes config file" is different than the IMEX config file (that is a different file in the same directory which we dont need to look at).

Reply:
OK. as long as the usage is consistent.

Reply:
do you like how it reads now (after recent push)